src/transformers/css2js.ts

```diff
@@ -0,0 +1,41 @@
+import * as CSS from "csstype";
+import { parseCss } from "../utils/parser";
+import { formatObject } from "../utils/formatter";
+
+// convert props
+// -webkit-flex becomes "-webkit-flex"
+// margin-bottom becomes marginBottom
+function convertCssPropToJs(prop: string): string {
+  return prop.startsWith("-")
+    ? `"${prop}"`
+    : prop.replace(/-([a-z])/g, (g) => g[1].toUpperCase());
+}
+
+function convertCssValueToJs(value: string): string {
+  return value.replace(/"/g, "'").replace(/(\d)px/g, "$1");
+}
+
+export function transform(css: string): string {
+  const declarations = parseCss(css);
+
+  const cssInJs = declarations.reduce<CSS.Properties>((acc, curr) => {
+    let prop: string = convertCssPropToJs(curr.prop);
+    let value: string | number = convertCssValueToJs(curr.value);
+    const numVal = Number(value);
+
+    return {
+      ...acc,
+      ...{
+        [prop]: isNaN(numVal) ? `"${value}"` : numVal,
+      },
+    };
+  }, {});
+
+  const objString: string = `{
+    ${Object.keys(cssInJs)
+      .map((key) => `${key}: ${cssInJs[key]}, `)
+      .join("\n")}
+  }`;
```

Comment:
I'm nitpicking here but I would prefer changing the logic a bit here.

Right now the `Declaration[]` is being reduced to an object. The issue I see with this is that an object is not an ordered data structure (at least semantically). But the order of declarations should be preserved in the output. Besides, I think `map` is more readable in this situation.

Instead of reducing, I'd suggest mapping each declaration to a string, or to an object like `{ prop, value }`. In js2jsx I chose to map to strings so it's easy to join them together, but how you like doing that is up to you.

Reply:
I chose to use an object because duplicate properties are ignored. 

Contrived example:
```css
div {
  width: 10px;
  height: 20px;
}

div {
  width: 20px;
  height: 100px;
}
```
Result:
```js 
{
  width: 20px;
  height: 100px;
}
```

The way you propose definitely improves the code. Let me know your thoughts

Reply:
@impulsivetomato That's a good point. I think the best of both worlds could be achieved with a [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map).

It might also be useful to add a test for this behavior.

Reply:
Hey @nimobeeren. I've pushed a fix using Map. Please have a look when you have a moment.

Reply:
@impulsivetomato Looks great, thank you!